### DISCUSSION TITLE
Using nicegui / catdesign BinauralBeats Generator

### DISCUSSION THREAD

**Discussion by ofenbach** [zauberzeug/nicegui#689] [Show and tell]
Showing a quick use case for nicegui/catdesign.

did this in 15min but works great:
https://github.com/ofenbach/BinauralBeats2

You are able to choose a base frequency and delta frequency to generate your own custom binaural beat or use  
presets. Just a quick demonstration of how easy it is to use nicegui in combination with catdesign.
Might improve this in the future. Just a quick coffee break project while other programs are compiling :)

<img width="1252" alt="SCREENSHOT 7" src="https://user-images.githubusercontent.com/29439003/229279130-493248bb-18a6-4889-a3cd-34d94777f633.png">

**Comment by stdusr** [comment]
What kind of thing is "catdesign", can you provide a link?

**Reply by falkoschindler** [reply]
Here it is: https://github.com/ofenbach/CatDesign

**Reply by stdusr** [reply]
Thanks!

**Reply by ofenbach** [reply]
Its a design system built on top of nice gui. Will improve with time
